src/test/java/org/apache/xml/security/utils/XMLUtilsTest.java

```diff
@@ -0,0 +1,56 @@
+package org.apache.xml.security.utils;
```

Comment:
Can you add the standard ASF license header here please as used by other files in the project?

Reply:
Sure, I will do that. 
I have a question: if is it okay to add "third-party library" com.sun.xml.messaging.saaj:saaj-impl to test this (I added that library because we had an issue with it using JDK 11), or is this a bit overkill?

Reply:
Yeah that's fine, it's just a test dependency